specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json

```diff
@@ -0,0 +1,1368 @@
+      "get": {
+        "tags": [
+          "BlobService"
+        ],
+        "operationId": "BlobServices_List",
+        "x-ms-examples": {
+          "ListBlobServices": {
+            "$ref": "./examples/BlobServicesList.json"
+          }
+        },
+        "description": "List blob services of storage account. It returns a collection of one object named default.",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- List blob services operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/BlobServiceItems"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/{BlobServicesName}": {
+      "put": {
+        "tags": [
+          "BlobService"
+        ],
+        "operationId": "BlobServices_SetServiceProperties",
+        "x-ms-examples": {
+          "PutBlobServices": {
+            "$ref": "./examples/BlobServicesPut.json"
+          }
+        },
+        "description": "Sets the properties of a storage account’s Blob service, including properties for Storage Analytics and CORS (Cross-Origin Resource Sharing) rules. ",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/BlobServicesName"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BlobServiceProperties"
+            },
+            "description": "The properties of a storage account’s Blob service, including properties for Storage Analytics and CORS (Cross-Origin Resource Sharing) rules."
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
No default responses here as @KrisBash  said.

Reply:
Here is what @KrisBash said. "Please add a default response that implements the common error contract. This should be on all API responses, but we can start with net new ones.". I will add a seperate PR to fix all existing apis.